pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml

```diff
@@ -0,0 +1,16 @@
+<component type="addon">
+  <id>org.cockpit-project.cockpit-selinux</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>SELinux</name>
+  <summary>
+    Security Enhanced Linux configuration and troubleshooting
```

Comment:
Does this actually install enough to make sure SELinux troubleshooting is usable on the system, beyond the Cockpit GUI? Are the dependencies of cockpit-selinux enough? Just checking ... because the app concept is about more than just Cockpit.

Reply:
I am not sure... We have a `Recommends: setroubleshoot-server` in cockpit-selinux.  Is that enough?

Reply:
You tell me. Given the ```summary```, by Installing the application I expect to do "SELinux Troubleshooting" and "SELinux configuration". Is ```Recommends: setroubleshoot-server``` enough for that? Are the relevant command line tools available and installed when that's installed? 

This isn't just about Cockpit, it's about bringing a feature (in this case SELinux config and troubleshooting) onto the server.

Reply:
We should change the `Recommends` to a `Requires`.

Reply:
Agreed, at least for the standalone `cockpit-selinux` package. On RHEL, where this is folded into `cockpit-system`, I think the Recommends is correct.